frontend/src/pages/destructionlist/Assignees.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  AttributeTable,
+  Body,
+  Button,
+  Errors,
+  Form,
+  FormField,
+  LabeledAttributeData,
+  Outline,
+  SerializedFormData,
+} from "@maykin-ui/admin-ui";
+import React, { FormEvent, useState } from "react";
+import { useActionData, useSubmit } from "react-router-dom";
+import { useAsync } from "react-use";
+
+import { listReviewers } from "../../lib/api/reviewers";
+import { AssigneesEditableProps, AssigneesFormProps } from "./types";
+
+export function AssigneesForm({
+  initialAssignees,
+  onClose,
+}: AssigneesFormProps) {
+  const errors = useActionData() || {};
+  const submit = useSubmit();
+
+  const { loading, value: availableReviewers = [] } = useAsync(async () => {
+    return await listReviewers();
+  }, []);
+
+  if (loading) return <Body>Loading...</Body>;
```

Comment:
- I think loading the date should happen in a loader (see destruction list create), the loading state can be obtained using a hook that will be in sync with the loader. Is there any reason not to use a loader here?
 - Maybe add some typographic element (e.g. P) to ensure the font is correct

Reply:
I thought that since the reviewers data is not needed unless the user wants to change the reviewers, it should be fetched only when the user clicks on the "edit the reviewers". But I can also load the data about users in the loader.

I thought I had added a "todo" note about the visual indication of loading, but clearly forgot :sweat_smile: That should replace the 'loading' message